comment_spell_check.py

```diff
@@ -366,6 +366,16 @@ def parse_args():
     return args
 
 
+def word_check(word):
```

Comment:
This function is a bit of an overkill and can be replaced by:
```
word.replace("'","").isidentifier()
```

Reply:
Change applied and commits squashed.